apps/common/src/python/mediawords/languages/en/en_stop_words.txt

```diff
@@ -4178,506 +1003,231 @@ there's
 there've
 thereafter
 thereby
+thered
 therefore
 therein
+therell
+thereof
+therere
 theres
+thereto
 thereupon
-thermal
+thereve
 these
 they
 they'd
 they'll
 they're
 they've
-thick
-thickness
+theyd
+theyll
+theyre
+theyve
 thin
 thing
 things
-think
-thinking
-thinks
 third
 thirty
 this
 thorough
 thoroughly
 those
+thou
 though
-thought
-thoughts
+thoughh
 thousand
-thousands
-threat
-threatened
-threatening
 three
-threw
-throat
+throug
 through
 throughout
-throw
-thrown
 thru
-thrust
-thursday
 thus
-thyroid
-tie
-tied
-tight
+til
 till
-time
-times
-tiny
 tip
-tire
-tired
-tissue
-title
+tis
+tj
+tk
+tm
+tn
 to
-toast
 today
-toes
 together
-told
-tomorrow
-tone
-tones
-tongue
-tonight
-tons
 too
 took
-tool
-tools
-tooth
 top
-torn
-tossed
-total
-totally
-touch
-touched
-tough
-tour
-tournament
 toward
 towards
-town
-towns
-trace
-track
-tractor
-trade
-traders
-trading
-tradition
-traditional
-traditions
-traffic
-tragedy
-tragic
-train
-trained
-training
-transfer
-transferred
-transformation
-transformed
-transition
-transportation
-trap
-travel
-traveled
-treat
-treated
-treatment
-tree
-trees
-trembling
-tremendous
-trend
-trends
-trial
-trials
-triangle
-tribute
+tp
+tr
 tried
 tries
-trim
-trip
-trips
-triumph
-troops
-trouble
-troubled
-troubles
-truck
-trucks
-true
-truly
-trust
-truth
+trillion
```

Comment:
I checked out a "biden" query and trillion stood out to me as a term that was in the old list and not in the new one. @ebndulue - do you have opinions about whether this should be included as a stop word?

Reply:
@ebndulue here's a link to the new stop words (probably easier to read than the github diff): https://github.com/mediacloud/backend/blob/91cb22c2e44ae6b3323d2459ce48f1296147386f/apps/common/src/python/mediawords/languages/en/en_stop_words.txt

Reply:
I'd get rid of "trillion" in the new list (and possibly other numbers) but IMHO something to keep in mind is that stopwords are very context-specific, and while "trillion" is not significant in many contexts, it would be in the context of the [Trillion Tree Campaign](https://en.wikipedia.org/wiki/Trillion_Tree_Campaign) coverage, so I'd rather keep the backend-filtered list very small (skip the "is", "not" etc.) and do most of the filtering on the frontend using the subject matter basis (perhaps a saved search could have a custom stopword list functionality?)